NwPluginAPI/Core/Player.cs

```diff
@@ -880,6 +880,16 @@ public void IntercomUnmute(bool revokeMute)
 		/// <param name="isFastRestart">Whether or not fast restart is enabled.</param>
 		public void Reconnect(float delay = 3f, bool isFastRestart = false) => Connection.Send(new RoundRestartMessage(isFastRestart ? RoundRestartType.FastRestart : RoundRestartType.FullRestart, delay, 0, true, false));
 
+		/// <summary>
+		/// Kills the player.
+		/// </summary>
+		/// <param name="reason">The reason for the kill</param>
+		/// <param name="cassieAnnouncement">The cassie announcement to make upon death.</param>
+		public void Kill(string reason, string cassieAnnouncement = "")
+		{
+			ReferenceHub.playerStats.KillPlayer(new CustomReasonDamageHandler(reason, -1f, cassieAnnouncement));
```

Comment:
```suggestion
			ReferenceHub.playerStats.KillPlayer(new UniversalDamageHandler(StandardDamageHandler.KillValue, DeathTranslations.Unknown));
```

`KillPlayer` is not public, because it's an internal function of `DealDamage`. Calling the internal function directly bypasses entire game logic, such as CASSIE announcements, granting spawn tickets or updating round summary stats. `DealDamage` is the public function that should be use instead.